odl/operator/default_ops.py

```diff
@@ -366,7 +369,118 @@ def __repr__(self):
 
     def __str__(self):
         """Return ``str(self)``."""
-        return "x * y"
+        return "x * {}".format(self.y)
+
+
+class PowerOperator(Operator):
+
+    """Take power of vector or scalar.
+
+    ``MultiplyOperator(n)(x) <==> x ** n``
+
+    Here, ``x`` is a `LinearSpaceVector` or `Field` element and
+    ``y`` is a number.
+    Hence, this operator can be defined either on a `LinearSpace` or on
+    a `Field`.
+    """
+
+    def __init__(self, domain, n):
+        """Initialize a MultiplyOperator instance.
```

Comment:
So we need some normalization here: do we use the class name in this summary or not? We should be stringent then, now it's mixed.